lib/protip/wrapper.rb

```diff
@@ -28,7 +28,9 @@ def respond_to?(name)
     def method_missing(name, *args)
       if (name =~ /=$/ && field = message.class.descriptor.detect{|field| :"#{field.name}=" == name})
         raise ArgumentError unless args.length == 1
-        set field, args[0]
+        attributes = {}.tap { |hash| hash[field.name] = args[0] }
+        assign_attributes attributes
```

Comment:
I can't tell if this is going to mess something up. Any thoughts?

I'm doing this so that the Resource can define it's own assign_attributes which goes through the "#{field_name}=" method. I want the assignments to go through "#{field_name}=" so that the fields get marked as dirty.